digi/xbee/models/address.py

```diff
@@ -42,7 +42,9 @@ class XBee16BitAddress(object):
     """
 
     PATTERN = "(0[xX])?[0-9a-fA-F]{1,4}"
```

Comment:
No need to separate pattern and the regular expression object as it already [contains the pattern](https://docs.python.org/3/library/re.html#regular-expression-objects). Also I use always use raw strings due to the [backslash plague](https://docs.python.org/3/howto/regex.html#the-backslash-plague). YMMV.

```
 __REGEXP = re.compile(r"(0[xX])?[0-9a-fA-F]{1,4}")

[...]

raise ValueError("Address must follow this pattern: " + cls.__REGEXP.pattern)
```

Reply:
I know, but constant `PATTERN` was already there and I do not want to break compatibility.